src/dense/interpolants.jl

```diff
@@ -14,10 +16,10 @@ end
 Hairer Norsett Wanner Solving Ordinary Differential Euations I - Nonstiff Problems Page 192
 """
 @def dp5pre0 begin
-    b10 = @evalpoly(Θ, 0, 1)
-    b20 = @evalpoly(Θ, 0, 1, -1)
-    b30 = @evalpoly(Θ, 0, 0, 1, -1)
-    b40 = @evalpoly(Θ, 0, 0, 1, -2, 1)
+    b10 = @evalpoly(Θ, NEGZERO, 1)
+    b20 = @evalpoly(Θ, NEGZERO, 1, -1)
+    b30 = @evalpoly(Θ, NEGZERO, NEGZERO, 1, -1)
+    b40 = @evalpoly(Θ, NEGZERO, NEGZERO, 1, -2, 1)
```

Comment:
IIUC @oscardssmith suggested to use e.g.
```suggestion
    b40 = Θ^2 * @evalpoly(Θ, 1, -2, 1)
```

Reply:
Ah, sorry, of course. That makes sense.

Reply:
That should be faster.